src/dotnet/commands/dotnet-add-p2p/Program.cs

```diff
@@ -16,34 +16,32 @@ public static int Run(string[] args)
             CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false)
             {
                 Name = "dotnet add p2p",
-                FullName = ".NET Add Project to Project (p2p) reference Command",
-                Description = "Command to add project to project (p2p) reference",
+                FullName = LocalizableStrings.AppFullName,
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
-                ArgumentSeparatorHelpText = "Project to project references to add"
+                ArgumentSeparatorHelpText = LocalizableStrings.AppHelpText
             };
 
             app.HelpOption("-h|--help");
 
             CommandArgument projectArgument = app.Argument(
-                "<PROJECT>",
-                "The project file to modify. If a project file is not specified," +
-                " it searches the current working directory for an MSBuild file that has" +
-                " a file extension that ends in `proj` and uses that file.");
+                $"<{LocalizableStrings.CmdProject}>",
```

Comment:
Why not just add the <> to CmdProject and here just use the property instead of having to interpolate?